app/controllers/root_controller.rb

```diff
@@ -1,4 +1,9 @@
+# frozen_string_literal: true
+
 class RootController < ApplicationController
-  def index
+  def index; end
+  def greeting
+    greeting = Greeting.all.sample.message
+    render json: greeting
   end
```

Comment:
- [ ] JSON should return a hash, not a string otherwise it will throw errors. We can return something like this:

  ```
  render json: { message: greeting }
  ```